protected/humhub/modules/space/models/AdvancedSettings.php

```diff
@@ -130,6 +130,24 @@ public function save(): bool
         $settings = $this->space->getSettings();
 
         $this->space->url = $this->url;
+
+        if ($this->space->isAttributeChanged('url')) {
```

Comment:
Instead of saving the URL, would it perhaps be possible to save the route of the `index` page?

Reply:
Only if this is possible to implement

Reply:
@luke- it's not only index page, so We need database migration to change all saved links to routes
A little bit change \humhub\modules\ui\menu\MenuLink and usages to have route also (currently it's only stored  href as string). This need to be done in core and in all modules which adds links to space left menu